src/pages/Account/Venue/VenueMapEdition/DraggableSubvenue.tsx

```diff
@@ -66,35 +76,39 @@ export const DraggableSubvenue: React.FC<PropsType> = (props) => {
         }}
         lockAspectRatio={lockAspectRatio}
       >
-        <div ref={drag} style={styles.dragContainer}>
-          <div
-            style={{
-              ...styles.resizeTab,
-              top: 0,
-              left: 0,
-            }}
-          />
-          <div
-            style={{
-              ...styles.resizeTab,
-              top: 0,
-              right: 0,
-            }}
-          />
-          <div
-            style={{
-              ...styles.resizeTab,
-              bottom: 0,
-              left: 0,
-            }}
-          />
-          <div
-            style={{
-              ...styles.resizeTab,
-              bottom: 0,
-              right: 0,
-            }}
-          />
+        <div ref={!isLoading ? drag : null} style={styles.dragContainer}>
```

Comment:
What are the changes to the `ref` here intended to do?

Reply:
When the room positions are being saved, they ref will die and the drag and drop will be disabled. I looked in the library on how to disabled the drag with a prop or ref but looks like it can't be disabled.